ext/re2/re2.cc

```diff
@@ -1418,15 +1461,19 @@ static VALUE re2_Replace(VALUE self, VALUE str, VALUE pattern,
     RE2::Replace(&str_as_string, StringValuePtr(pattern),
         StringValuePtr(rewrite));
 
-    return ENCODED_STR_NEW2(str_as_string.data(), str_as_string.size(),
-        pattern);
+    return ENCODED_STR_NEW(str_as_string.data(), str_as_string.size(),
+        "UTF-8");
```

Comment:
@stanhu will this (and the change to `GlobalReplace` below) affect GitLab or are you only using UTF-8 strings with re2?

Reply:
As far as I can tell, GitLab is only using UTF-8 strings with re2.